Fireworks/Geometry/python/dumpRecoGeometry_cfg.py

```diff
@@ -171,16 +190,33 @@ def recoGeoLoad(score):
 
 options.parseArguments()
 
-
-
-
-process = cms.Process("DUMP")
+from Configuration.PyReleaseValidation.upgradeWorkflowComponents import upgradeProperties as properties
+# Determine version_key based on the value of options.tag
+if options.tag == "2026" or options.tag == "MaPSA":
+   prop_key = 2026
+   version_key = options.tag + options.version
+elif options.tag == "2017": #or options.tag == "2021": (this leads to crashes in tests ?)
```

Comment:
@cms-sw/geometry-l2 if I add here the 2021 part (which makes use the era `Run3`) I get a failure:

```
----- Begin Fatal Exception 27-Jun-2024 14:17:04 CEST-----------------------
An exception of category 'NoProductResolverException' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'p'
   [2] Prefetching for module DumpFWRecoGeometry/'dump'
   [3] Prefetching for EventSetup module FWRecoGeometryESProducer/''
   [4] Prefetching for EventSetup module CaloGeometryBuilder/''
   [5] Calling method for EventSetup module EcalPreshowerGeometryEPdd4hep/'EcalPreshowerGeometryEP'
Exception Message:
No data of type "cms::DDCompactView" with label "" in record "IdealGeometryRecord"
 Please add an ESSource or ESProducer to your job which can deliver this data.
----- End Fatal Exception -------------------------------------------------
```

while that's not the case when no era is used. Is something missing in the GlobalTags ?